api/models/User.js

```diff
@@ -20,17 +23,14 @@ module.exports = {
     emailConfirmed: {
       type      : 'boolean',
       defaultsTo: false
-    },
-
-    toJSON: function() {
-      var values = this.toObject();
-
-      delete values.password;
-
-      return values;
     }
   },
 
+  customToJSON: function() {
+  // Return a shallow copy of this record with the password.
+    return _.omit(this, ['password' ])
```

Comment:
Why not just call `toObject`?

Reply:
It is like that in the sails doc

https://github.com/balderdashy/sails-docs/blob/1.0/upgrading/To1.0.md

The .toObject() instance method is no longer added to every record. When implementing customToJSON for a model, be sure to clone the record using _.omit(), _.pick() or _.clone().

Reply:
It is for sails < 1 however. I know bumping it to version 2.0 makes it a breaking change but do we still want to support sails 0.12? @RWOverdijk

Reply:
But I thought that @RWOverdijk suggested to create a preview version for sails V1
To be checked with him